### ISSUE TITLE
CompositeAggregatorTests.testWithKeywordLongAndMissingBucket test failure

### ISSUE THREAD

**Issue by dreamer-89** [opensearch-project/OpenSearch#2012]
Assertion failure on bucket size
```
REPRODUCE WITH: ./gradlew ':server:test' --tests "org.opensearch.search.aggregations.bucket.composite.CompositeAggregatorTests.testWithKeywordLongAndMissingBucket" -Dtests.seed=15968AF9CDC349BA -Dtests.security.manager=true -Dtests.jvm.argline="-XX:TieredStopAtLevel=1 -XX:ReservedCodeCacheSize=64m" -Dtests.locale=el-GR -Dtests.timezone=Australia/ACT -Druntime.java=17

org.opensearch.search.aggregations.bucket.composite.CompositeAggregatorTests > testWithKeywordLongAndMissingBucket FAILED
    java.lang.AssertionError: expected:<2> but was:<3>
        at __randomizedtesting.SeedInfo.seed([15968AF9CDC349BA:F4A96A9EAA1D27F5]:0)
        at org.junit.Assert.fail(Assert.java:89)
        at org.junit.Assert.failNotEquals(Assert.java:835)
        at org.junit.Assert.assertEquals(Assert.java:647)
        at org.junit.Assert.assertEquals(Assert.java:633)
        at org.opensearch.search.aggregations.bucket.composite.CompositeAggregatorTests.lambda$testWithKeywordLongAndMissingBucket$89(CompositeAggregatorTests.java:1095)
        at org.opensearch.search.aggregations.bucket.composite.CompositeAggregatorTests.executeTestCase(CompositeAggregatorTests.java:2712)
        at org.opensearch.search.aggregations.bucket.composite.CompositeAggregatorTests.testSearchCase(CompositeAggregatorTests.java:2660)
        at org.opensearch.search.aggregations.bucket.composite.CompositeAggregatorTests.testWithKeywordLongAndMissingBucket(CompositeAggregatorTests.java:1087)
```

Gradle log: https://fork-jenkins.searchservices.aws.dev/job/OpenSearch_CI/job/PR_Checks/job/Gradle_Check/2122/artifact/gradle_check_2122.log

**Comment by penghuo** [comment]
Does it same as #1988? I suppose it already been fixed by #2005.

**Comment by penghuo** [comment]
@dblock Could we close this issue? It already been fixed by #2005.
